Makefile.in

```diff
@@ -135,51 +135,17 @@ dep depend fastdep: $(DEPEND_FILE)
 ################################################################################
 dev: distclean autotools all
 
-################################################################################
-# DEV-GIT, BUILD with developer flags                                          #
-################################################################################
-dev-git: distclean autotools-git all
-
-
-################################################################################
-# GIT, BUILD with developer flags                                              #
-################################################################################
-build-commit-git: distclean set-version-git all
-
-################################################################################
-# CURRENT, BUILD current svn trunk.                                            #
-################################################################################
-current: distclean svn autotools all
-
-svn:
-	svn update
-
 autotools:
-	@sed -e 's/.\/commit-version.sh/.\/version.sh/g' configure.ac > configure.ac.tmp && mv -f configure.ac.tmp configure.ac
+	@sed -e 's/.\/version.sh/.\/version.sh/g' configure.ac > configure.ac.tmp && mv -f configure.ac.tmp configure.ac
```

Comment:
The pattern in this sed command doesn't actually appear to change anything now - it's replacing ./version.sh with ./version.sh, so we could just delete this line completely? (same applies to other 2 instances)